clients/src/test/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumerTest.java

```diff
@@ -419,6 +422,35 @@ public void testWakeupAfterNonEmptyFetch() {
         assertThrows(WakeupException.class, () -> consumer.poll(Duration.ZERO));
     }
 
+    @Test
+    public void testCommitInCommitCallback() {
```

Comment:
We're resting here the commit on rebalance callback, so name maybe `testCommitInRebalanceCallback` ?

Reply:
oops